src/dc_aheader.rs

```diff
@@ -146,7 +147,7 @@ pub unsafe fn dc_aheader_set_from_string(
                     .offset(strspn(p, b"\t\r\n ;\x00" as *const u8 as *const libc::c_char) as isize)
             }
             *after_attr_name = '\u{0}' as i32 as libc::c_char;
-            if !(0 == add_attribute(aheader, beg_attr_name, beg_attr_value)) {
+            if add_attribute(aheader, beg_attr_name, beg_attr_value) {
```

Comment:
Forgot to mention this in the PR, but only used locally in `dc_aheader.rs` file.